packages/components/forms/src/radio/Radio.mdx

```diff
@@ -84,11 +84,11 @@ You can use the Radio as an uncontrolled input, for that you can:
 </React.Fragment>
 ```
 
-### Using with RadioGroup
+### Using with Group
 
 import { FormControl } from '@contentful/f36-forms';
 
-We also provide the `RadioGroup` component that helps when using multiple radio, you can pass the spacing value and some common properties to be replicated to the radios.
+We also provide the `Radio.Group` component that helps when using multiple radio, you can pass the spacing value and some common properties to be replicated to the radios.
```

Comment:
Same as for checkboxes, maybe we could rewrite it so it's really clear that those props will be passed to inputs in the group.